### DISCUSSION TITLE
Support for Kustomize v4.1.0

### DISCUSSION THREAD

**Discussion by alexis-wefight** [argoproj/argo-cd#6656] [Q&A]
Hi,

I would like to apply patch merge strategy using Kustomize with ArgoCD on CRD. Kustomize v4.1.0 provide a new feature to define the CRD openapi schema, to apply the merge strategy (especially to merge a list). 

However it seems the Kustomize default version is v3.9.4 for ArgoCD 2.0. I tried to update the version used according the [doc](https://argoproj.github.io/argo-cd/user-guide/kustomize/) but I have this error:

```
rpc error: code = Unknown desc = Manifest generation error (cached): fork/exec /custom-tools/kustomize_4_2_0: no such file or directory
```

Is it possible to use Kustomize v4.1.0 with the latest version of ArgoCD?

Thank you.

**Comment by alexis-wefight** [comment]
Any suggestion?

**Reply by Bogay** [reply]
Have you tried install Kustomize v4.1.0 on your argocd-repo-server? ([doc](https://argo-cd.readthedocs.io/en/stable/operator-manual/custom_tools/))
